_includes/header.html

```diff
@@ -13,12 +13,12 @@
     <input id="q" placeholder="🔎 search">
     <ul id="navmenu">
       <li>
-        <a href="/" id="home-menu" {% if page.menu=='home' %} class="active" {% endif %}>
+        <a id="home-menu" {% if page.menu=='home' %} class="active" {% else %} href="/" {% endif %}>
           {{ site.data[page.lang].menu.home }}
         </a>
       </li>
       <li id="getting-started-menu" class="submenu">
-        <a href="/{{ page.lang }}/starter/installing.html" {% if page.menu=='starter' %} class="active" {% endif%}>
+        <a {% if page.menu=='starter' %} class="active" {% else %} href="/{{ page.lang }}/starter/installing.html" {% endif%}>
```

Comment:
```suggestion
        <a {% if page.menu=='starter' %} class="active" {% else %} href="/{{ page.lang }}/starter/installing.html" {% endif%}>
```
I do not support removing `href` if link is active. It is bad practice. This breaks a11y features. Not having an href is semantically incorrect for anchor tags